src/app/pages/invoices/components/options/options.component.html

```diff
@@ -1,43 +1,37 @@
-<ion-list>
-    <ion-item button lines="none">
+<ion-list *transloco="let t; read: 'invoices'">
+    <!-- <ion-item button lines="none">
         Open details
         <ion-icon color="dark" slot="start" name="document-text-outline"></ion-icon>
-        <!-- <ion-icon color="primary" name="open-outline" slot="start"></ion-icon> -->
-    </ion-item>
+    </ion-item> -->
 
     <ion-item button lines="none" (click)="openInvoice()">
-        Open PDF
+        {{ t("Open PDF") }}
         <ion-icon color="primary" slot="start" name="newspaper-outline"></ion-icon>
-        <!-- <ion-icon color="primary" name="open-outline" slot="start"></ion-icon> -->
     </ion-item>
 
-    <ion-item button lines="none" (click)="print()">
+    <!-- <ion-item button lines="none" (click)="print()">
         Print invoice
         <ion-icon color="secondary" slot="start" name="print-outline"></ion-icon>
-        <!-- <ion-icon color="primary" name="open-outline" slot="start"></ion-icon> -->
-    </ion-item>
+    </ion-item> -->
 
-    <ion-item button lines="none">
+    <!-- <ion-item button lines="none">
         Download PDF & XML
         <ion-icon name="cloud-download-outline" slot="start" color="tertiary"></ion-icon>
-        <!-- <ion-icon color="primary" name="create-outline" slot="start"></ion-icon> -->
-    </ion-item>
+    </ion-item> -->
 
     <ion-item button lines="none" (click)="proofPaymentDoc()">
-        Proof of Payment
+        {{ t("Proof of Payment") }}
         <ion-icon name="cloud-upload-outline" slot="start" color="warning"></ion-icon>
     </ion-item>
 
     <ion-item button lines="none" *ngIf="invoice.needsAttention" (click)="openFixInvPage()">
-        Fix inventory
+        {{ t("Fix Inventory") }}
         <ion-icon color="dark" slot="start" name="create-outline"></ion-icon>
-        <!-- <ion-icon color="primary" name="open-outline" slot="start"></ion-icon> -->
     </ion-item>
 
-    <ion-item button lines="none">
-        Cancel invoice
+    <ion-item button lines="none" (click)="cancelInvoice()" *ngIf="invoice.status !== 'cancelled' && invoice.status !== 'canceled'">
```

Comment:
Should I leave this cancel functionality here?

Reply:
Comment out for now, need to add implementation

Reply:
Alright, will comment out the button for now.